### DISCUSSION TITLE
How to use `defaultBackgroundImage` in content - articles, lists, not connected to homepage?

### DISCUSSION THREAD

**Discussion by tetricky** [nunocoracao/blowfish#1735] [Q&A]
Thank you very much for blowfish, and blowfish-tools. I am far from an expert with web development, but this makes a very satisfactory platform extremely accessible.

I am not well versed in Hugo either...I'm increasing my experience through exposure via blowfish. This is likely to be a stupid question. My apologies.

I have a `defaultBackgroundImage` set globally, along with a `homepageImage` set in [homepage] with `layout` set to `background` This works well for the homepage.

I have content, that is not listed on the homepage, but rather accessible by menu's (all working well). I would like to have the background image displayed as the default background in other content. The structure and addition of new posts to these menu lists work perfectly.

I cannot seem, whatever settings I have tried globally, to get any form of background image displaying on any of these posts within menus (not directly linked to the homepage).

Am I using blowfish wrongly, or overlooking something simple? Is there a way to achieve what I am looking to do?

I'm building this in a local environment using blowfish-tools (and manually), also using github actions to build for gh pages, and a static hosting environment (pushed to from either a local build, or gh actions). I'm getting consistent behaviour whatever the backend.

**Comment by bootlegcoal** [comment]
Hi @tetricky , did you ever figure this out? Presumably blowfish.page is configured in a way that we can do as well? Idk.

**Comment by tetricky** [comment]
No, I'm afraid I didn't ever figure it out. I'm just making do without.

**Comment by RoozbehSh** [comment]
Hi! I had the same problem and just figured it out. In the `params.toml`, the article section, make the following changes:
```
[article]
  showHero = true
  heroStyle = "background"
  layoutBackgroundBlur = true
```
